src/users_router.rs

```diff
@@ -344,6 +344,7 @@ describe! setup_tests {
                 assert!(false);
             }
         };
+        remove_test_db();
```

Comment:
When I do that I seem to get a test failure. And I haven't figured out how to view the generated code by the macro - to see what was happening - so I chose the easy way.